src/discussions/posts/post/Post.jsx

```diff
@@ -0,0 +1,111 @@
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { faQuestionCircle, faStar as faEmptyStar } from '@fortawesome/free-regular-svg-icons';
+import {
+  faCircle, faComments, faFlag, faStar as faSolidStar, faThumbtack,
+} from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Image } from '@edx/paragon';
+import { Routes } from '../../../data/constants';
+import messages from './messages';
+
+function Post({ post, intl }) {
+  return (
+    <Link
```

Comment:
I this isn't exactly in the scope here, but clicking on a post, no matter in which page, will make the page navigate to the posts view (or at least change the `Nav` to that).

![nav_issue](https://user-images.githubusercontent.com/27893385/92999338-762cc880-f4f6-11ea-8f37-9a28478ed084.gif)

This is likely due to the page structure and will need to be reworked in a future iteration.

CC @xitij2000

Reply:
I think this is OK to leave for another PR. This is because there isn't an unambiguous way to determine which view we are on from the current URL. Chances are this will need either a change in the URL structure or we can save this info to redux.

Reply:
:+1: